tests/e2e/pinecone_e2e/populate_dbs.py

```diff
@@ -0,0 +1,148 @@
+#  Copyright (c) "Neo4j"
+#  Neo4j Sweden AB [https://neo4j.com]
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#      https://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import hashlib
+import json
+import os.path
+
+from neo4j import GraphDatabase
+from pinecone import Pinecone, ServerlessSpec
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+def populate_dbs(neo4j_driver, pc_client, index_name="jeopardy"):
+    neo4j_objects, pc_question_objs = build_data_objects()
+
+    pc_client.create_index(
+        name=index_name,
+        dimension=384,
+        metric="cosine",
+        spec=ServerlessSpec(cloud="aws", region="us-east-1"),
+    )
+
+    # Populate Pinecone
+    populate_pinecone(pc_client, pc_question_objs, index_name)
+
+    # Populate Neo4j
+    populate_neo4j(neo4j_driver, neo4j_objects)
+
+
+def populate_neo4j(neo4j_driver, neo4j_objs):
```

Comment:
Since neo4j population function is being used both in the pinecone and weaviate population files, maybe it's worth pulling to a separate file.